tpm/constants.go

```diff
@@ -162,6 +164,35 @@ const (
 	authPrivUseOnly byte = 0x03
 )
 
+// Locality Selection
+
+const (
+	locZero byte = 1 << iota
+	locOne
+	locTwo
+	locThree
+	locFour
+)
+
+// NV Permissions and Operations
+
+const (
+	nvPerPPWrite    uint32 = 0x00000001
+	nvPerOwnerWrite uint32 = 0x00000002
+	nvPerAuthWrite  uint32 = 0x00000004
+	nvPerWriteAll   uint32 = 0x00000800
+	// Warning: The Value 0x00001000 is defined in the spec as TPM_NV_PER_WRITEDEFINE,
+	// but it is not included directly in this code because it locks the given NV Index permanently
```

Comment:
Please reflow the text of this comment to even out the lines to match line length of the rest of the code.